go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/verrazzano/verrazzano-operator
 require (
 	github.com/Jeffail/gabs/v2 v2.3.0
 	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
+	github.com/gordonklaus/ineffassign v0.0.0-20200809085317-e36bfde3bb78 // indirect
```

Comment:
ideally, we should not check indirects in - you can run `go mod tidy` before checkin and it should get rid of these ("should" be the operative word)

Reply:
done